base/challenge-skeleton/README.md

```diff
@@ -0,0 +1,40 @@
+# Quickstart guide to writing a challenge:
+
+* Put your challenge binary under challenge/image/ and update challenge/image/Dockerfile.
+* By default /chroot/home/user/chal will be run. The command line can be changed in challenge/runtime-files/nsjail.cfg.
+* Try the challenge locally with `make test-docker`.
+* If you have a cluster ready, try it remotely with `make start && make port-forward`.
+
+## healthcheck
+
+* The healthcheck needs to run a webserver on port 8080 that responds to /healthz requests.
+ * 200 if the challenge is healthy, otherwise an error (e.g. 400).
```

Comment:
Same as above, I accidentally deployed a web server in the challenge container in port 8080. Now I realize this also would make the healthcheck think it's working.

Maybe we should make it require responding with OK on /healthz.